src/utils/hooks/api/use-get-collateral-currencies.tsx

```diff
@@ -0,0 +1,15 @@
+import { CollateralCurrencyExt, getCorrespondingCollateralCurrencies } from '@interlay/interbtc-api';
+import { useQuery, UseQueryResult } from 'react-query';
+
+const getCollateralCurrencies = async (): Promise<Array<CollateralCurrencyExt>> => {
+  // TODO: Replace with call to lib that returns all collateral currencies when ready on lib side.
+  const nativeCollateralCurrencies = getCorrespondingCollateralCurrencies(window.bridge.getGovernanceCurrency());
+  const foreignCurrencies = await window.bridge.assetRegistry.getForeignAssets();
+  return [...nativeCollateralCurrencies, ...foreignCurrencies];
```

Comment:
Blocker - the code assumes all foreign assets are enabled as vault collateral. Therefore, this code needs to be changed to call lib function that returns only enabled collateral currencies (both native and foreign) - already planned with @bvotteler.